schema/buildings/defs.yaml

```diff
@@ -20,6 +20,11 @@ shapeContainer:
       description: >-
         The height of the bottom part of building in meters. Used if a building or part of building starts above the ground level.
       type: number
+    min_floor:
+      description: >-
+        The "start" floor of this building or part. This is the analog of "min_height" but is sometimes populated when the
+        start height is unknown.
+      type: integer
```

Comment:
Any validation desired? *e.g.* `minimum` of zero, or `minimum` of one?

How are basements typically handled? Or floors that are not labeled with a number, P2, P3, B, B2, sub-basement, L, mezzanine and so forth?

Reply:
I took a look at OSM. There are ~137000 ways that have a building:min_level. Less than 1000 are less than zero. Given that - I think we should probably make this greater than 0 and omit the value when it's less than zero. This includes our understanding that this number should be positive anyway.

Basements and other underground levels (like parking levels) are a separate field in OSM that's not currently handled in Overture. But it does not affect the value of this field.

Reply:
Added exclusiveMinimum.